api/src/models/vote.ts

```diff
@@ -0,0 +1,21 @@
+import mongoose from 'mongoose';
+
+const voteSchema = new mongoose.Schema({
+  userID: {
+    type: String,
+    ref: 'User',
+  },
+  reviewID: {
+    type: String,
+    ref: 'Review',
+  },
+  timestamp: {
+    type: Date,
+    required: true,
+    default: Date.now,
+  },
```

Comment:
should also have a score attribute with value being 1 or -1 (1 being upvote, -1 being downvote)

Reply:
upvoting and downvoting is also broken, i think this is the cause